app/controllers/home_controller.rb

```diff
@@ -13,9 +16,32 @@ def administration; end
 
   def recruiting; end
 
+  def civil_servant_faq; end
+
   def agb; end
 
   def privacy_policy; end
 
   def about_us; end
+
+  private
+
+  def load_faqs
+    @faqs = Faq.all
+  end
+
+  # :reek:FeatureEnvy
+  def load_structured_faqs
```

Comment:
Why is that in the controller? I'd move that to a helper, then you even lose an instance variable

Reply:
True